### ISSUE TITLE
basic_json_dump.py yields: TypeError: 'list' object is not an iterator 

### ISSUE THREAD

**Issue by Mark-L6n** [kensho-technologies/qwikidata#35]
When using the example code from https://github.com/kensho-technologies/qwikidata/tree/develop/examples/basic_json_dump.py (which appears to be the same as the [docs](https://qwikidata.readthedocs.io/en/stable/readme.html#json-dump)), I get the following error messages:

```
File "test-qwikidata.py", line 56, in <module>
    dump_entities_to_json(politicians, out_fname)
File "path/to/py35/lib/python3.5/site-packages/qwikidata/utils.py", line 31, in dump_entities_to_json
    ent = next(entities, None)
TypeError: 'list' object is not an iterator
```

Used dump `latest-all.json.bz2` downloaded 11/4/22 from [here](https://dumps.wikimedia.org/wikidatawiki/entities/)  
Occurs in both python 3.5 and 3.10.

Is there a way around this?

**Comment by galtay** [comment]
hey @Mark-L6n , thanks for opening an issue ... looks like [this PR](https://github.com/kensho-technologies/qwikidata/pull/34) introduced this issue ... I can take a look tonight

**Comment by galtay** [comment]
to check if my hunch is correct you could try passing `iter(politicians)` to the `dump_entities_to_json` function

**Comment by Mark-L6n** [comment]
Your hunch is correct, that works, thanks!

**Comment by galtay** [comment]
fixed in #36 and new version pushed to PyPI, thanks again for the report! 
https://pypi.org/project/qwikidata/
